src/detect-flowbits.c

```diff
@@ -69,6 +73,37 @@ void DetectFlowbitsRegister (void)
     DetectSetupParseRegexes(PARSE_REGEX, &parse_regex);
 }
 
+static char **StrSplit(char *arrptr, const char *delim)
+{
+    char **strarr = NULL;
+    char *token, *saveptr;
```

Comment:
I don't know if its strictly necessary, but I'd sleep better if `saveptr` was initialized to `NULL` here.